gpp_parsed_consent.go

```diff
@@ -0,0 +1,41 @@
+package iabconsent
+
+import (
+	"encoding/base64"
+	"fmt"
+
+	"github.com/pkg/errors"
+)
+
+type GppHeader struct {
+	Type     int
+	Version  int
+	Sections []int
+}
+
+// ParseGppHeader parses the first (and required) part of any GPP Consent String.
+// It is used to read the Type, Version, and which sections are contained in the following string(s).
+// Format is:
+// Type	    Int(6)	Fixed to 3 as “GPP Header field”
+// Version	Int(6)	Version of the GPP spec (currently 1)
```

Comment:
instead of "currently" can you include the date jan 2023? like Version 1 as of Jan 2023

Reply:
Updated!